Sources/MobileCoinClient.swift

```diff
@@ -193,17 +194,21 @@ public final class MobileCoinClient {
         memoType: MemoType = .recoverable,
         amount: Amount,
         fee: UInt64,
+        rng: MobileCoinRng? = nil,
```

Comment:
```suggestion
        rng: MobileCoinRng = defaultRng,
```

Reply:
@the-real-adammork  So we can do this, but we have to make the defaultRng attribute of MobileCoinClient public static for it to be used as a default param value.  Might be some other option - will explore further.  For now will update to make it work and let you take a look.